packages/teams-js/test/private/files.spec.ts

```diff
@@ -398,21 +399,23 @@ describe('files', () => {
       await utils.initializeWithContext('content');
```

Comment:
Can you please add `expect.assertions(1)` (or whatever the correct number is) to ensure that the test validates the `expect` in the catch block?